src/Compilation.zig

```diff
@@ -1048,6 +1048,18 @@ pub fn create(gpa: Allocator, options: InitOptions) !*Compilation {
             break :blk buf.items[0 .. buf.items.len - 1 :0].ptr;
         } else null;
 
+        if (options.verbose_llvm_cpu_features) {
+            if (llvm_cpu_features) |cf| {
+                std.debug.getStderrMutex().lock();
+                defer std.debug.getStderrMutex().unlock();
+                const stderr = std.io.getStdErr().writer();
+                nosuspend try stderr.print("compilation: {s}\n", .{options.root_name});
+                nosuspend try stderr.print("  target: {s}\n", .{try options.target.zigTriple(arena)});
+                nosuspend try stderr.print("  cpu: {s}\n", .{options.target.cpu.model.name});
+                nosuspend try stderr.print("  features: {s}\n", .{cf});
```

Comment:
In case of an error, I think we should just skip subsequent prints, leave the block and keep going with the compilation, just like we do in `Compilation.dump_argv`.

Reply:
done and rebased